Hunting Queries/SecurityEvent/ProcessEntropy.txt

```diff
@@ -14,6 +14,7 @@
 //
 // Tactics: #Execution
 //
+// May need to reduce the number of days if the environment is very large.  Try 3-5 days, but less accuracy.
 let end = startofday(ago(7d));
```

Comment:
Shouldn't this be startofday(now()) ?
Or is it really meant to be the 7 days before the current 7 days?

Reply:
Yes...typo.  Thanks for catching. :)